### ISSUE TITLE
Add a dropmarker to Quick Format citation bubbles

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#2712]
https://twitter.com/grace_lao/status/1549544846913933313

I've been thinking for a while that we need to make the popup more obvious. People in the forums pretty regularly have no idea that there's more to the dialog.

We probably want a light-blueish triangle on the right-hand side. Will slightly ruin the asymmetry of the bubbles, but I think it's worth it for the increased usability.

**Comment by dstillman** [comment]
https://twitter.com/WorldImagining/status/1563156252729180163

**Comment by yexingsha** [comment]
Since many people aren't aware they can click on bubbles to customize citations, let's add dropmarkers to them:

<img width="1280" alt="Image" src="https://github.com/user-attachments/assets/5ef05bb3-2d5a-4aaf-8929-9abb5c9bba22" />

The dropmarker uses `chevron-6.svg`. The gap between the text and the dropmarker is 4px. 

Hovering over a bubble will no longer show the `X` button and mask, since people can always delete a bubble using either the keyboard or the `Remove` button in the popup.

**Comment by yexingsha** [comment]
> But one question I have is this: isn't not knowing that a bubble is clickable a one-time problem? But once you find it out, then you just know. Then, after you do discover it, the dropmarkers would be no longer useful, and you'd rather have the X button that can actually perform an action in its place (regardless how often you would use it)?

I agree this is the case, but it seems to be a problem for a so many people that the utility of dropmarkers might outweigh the utility of the occasionally relevant X button.

> what if we just ... open the itemDetails popup for the user ourselves at some point? Maybe, if one opens the citation dialog for the very first time, as soon as they add their first item in the citation, instead of refocusing the input, we open the bubble's popup? That would be the most direct way to indicate that there is a popup associated with a bubble .

I think the inconsistency could be confusing rather than illuminating, making people wonder why this popup automatically appeared before and not now. And it can run into the same problem as first-run guidance panels, which is that people sometimes just don't pay attention. @dstillman what do you think?

**Comment by dstillman** [comment]
> Does it feel slightly off to be explicitly removing an existing feature?

I mean, I wouldn't really call it an "existing feature" — it's been out in a beta for a week. People have gotten by without the X for many years, and I don't recall this being a source of confusion. Conversely, people not realizing that you could and should customize citations via Quick Format has been a major problem for years, and I don't think the new button style really does much to make that clearer.

> It's quite possible that people remove bubbles way less frequently than I do during testing and such, so maybe the X is actually not that valuable.

Yes. More importantly, they almost certainly remove bubbles far less than they (should) customize citations. And there are two alternative ways to remove citations within the dialog. The alternative to not realizing you can customize citations is getting confusing prompts and breaking citation updates in your document.

I also would disagree that this is a one-time thing. Word processor integration isn't necessarily someone uses every day — people could easily go months between working on something that needs it. So I think it's plausible that someone could return to this dialog and not remember that there's a way to customize citations here.

> Maybe, if one opens the citation dialog for the very first time, as soon as they add their first item in the citation, instead of refocusing the input, we open the bubble's popup

Yeah, I think that would be confusing. If anything, we should just restore the guidance panel, which also explains the locator typing trick. (But yes, most people won't actually pay attention to either, so just making the UI clear is a better long-term solution.)

**Comment by abaevbog** [comment]
Gotcha! All of this makes a lot of sense, thank you for the comments and explanation. 

> More importantly, they almost certainly remove bubbles far less than they (should) customize citations.

This is an important point - I will keep it in mind. If there is confusion that leads people to essentially not use the dialog properly, and a dropmarker can alleviate that, it 100% outweighs whatever (if any) benefits of the `X` button there may have been.

I added a linked PR with the dropmarker instead of the `X` button and minor cleanup. I think it's looking quite nice. I quite like not having the mask effect and the `X` button appearing only when the bubble is hovered. A dropmarker that's just always there looks more consistent. @yexingsha let me know how it's looking to you!
